src/lib/components/Dropdown/Dropdown.tsx

```diff
@@ -86,8 +91,25 @@ const DropdownComponent: FC<DropdownProps> = ({ children, className, dismissOnCl
     [children, theme],
   );
 
-  const TriggerWrapper: FC<ButtonProps> = ({ children }): JSX.Element =>
-    inline ? <button className={theme.inlineWrapper}>{children}</button> : <Button {...buttonProps}>{children}</Button>;
+  const TriggerWrapper: FC<TriggerWrapperProps> = ({ children, getWidth }): JSX.Element => {
+    const inlineRef = useRef<HTMLButtonElement | null>(null);
```

Comment:
Is `inlineRef` needed at all, or can we just use the one `ref` for either case? Since you can't have both an inline dropdown and normal dropdown simultaneously, I'm wondering if React stops you from just passing the same ref to both components via L104, L108